tests/lib/rules/eqeqeq.js

```diff
@@ -31,7 +31,14 @@ ruleTester.run("eqeqeq", rule, {
         { code: "null == a", options: ["smart"] },
         { code: "a == null", options: ["smart"] },
         { code: "null == a", options: ["allow-null"] },
-        { code: "a == null", options: ["allow-null"] }
+        { code: "a == null", options: ["allow-null"] },
+        { code: "a == null", options: ["always", {null: "ignore"}] },
+        { code: "a != null", options: ["always", {null: "ignore"}] },
+        { code: "a !== null", options: ["always", {null: "ignore"}] },
+        { code: "a === null", options: ["always", {null: "always"}] },
+        { code: "a !== null", options: ["always", {null: "always"}] },
+        { code: "a == null", options: ["always", {null: "never"}] },
+        { code: "a != null", options: ["always", {null: "never"}] }
```

Comment:
I don't know if it might be worth having some tests for `null === null` and `null == null` and similar to make sure they still behave correctly? (Maybe not, but it wouldn't be hard to add, no?)